bundles/http/Citrix/7150aa4a-3bcd-4a73-a15c-893249515184/101b58cb-1023-48a1-9aca-4b64c2dbfd68.js

```diff
@@ -0,0 +1,402 @@
+const pageSize = 100
+
+function fullSync ({ client, dataStore, integrationParameters }) {
+  const credentials = getCredentials(client, integrationParameters)
+
+  syncProjects({client, dataStore, credentials})
+  syncUsers({client, dataStore, credentials})
+  syncViews({ client, dataStore, credentials })
+  syncWorkbooks({client, dataStore, credentials})
```

Comment:
All of these functions should be in `return Promise.all([...])` because then MAS server cannot know when it will be finished.